doc/classes/CompressedTexture2D.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CompressedTexture2D" inherits="Texture2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A [code].ctex[/code] texture.
+		VRAM-compressed texture with 2 dimensions.
```

Comment:
Indeed, I forgot that CompressedTexture is formerly the name of StreamTexture. It's confusing that it can also be used for uncompressed textures though.